assets/src/components/routeLadder.tsx

```diff
@@ -177,6 +177,27 @@ export const groupByPosition = (
   }
 }
 
+const runNotSharedByAnotherVehicle = (vehiclesAndGhosts: VehicleOrGhost[]) => (
+  vehicle: Vehicle
+): boolean => {
+  if (vehicle.runId === null) {
+    return false
+  }
+
+  const otherVehicles = onlyVehicles(vehiclesAndGhosts).filter(
+    ({ id }) => id !== vehicle.id
+  )
+  const otherRunIds = runIds(otherVehicles)
+
+  return !otherRunIds.includes(vehicle.runId)
+}
+
+const onlyVehicles = (vehiclesAndGhosts: VehicleOrGhost[]): Vehicle[] =>
```

Comment:
Is this needed? Ghosts wouldn't be made with a matching run, and even if there was one, you probably wouldn't want to draw the virtual ghost.

Reply:
Yeah, I guess that's true.